ddtrace/internal/ci_visibility/_api_client.py

```diff
@@ -80,6 +87,13 @@ class EarlyFlakeDetectionSettings:
     faulty_session_threshold: float = 30.0
 
 
+@dataclasses.dataclass(frozen=True)
+class UniqueTestsData:
```

Comment:
Where is this used? :eyes:

Reply:
It is not, I thought it'd be useful and then decided not to use it and then (I think I can say "as usual", at this point), I forgot to remove it.